client/src/components/molecules/FileList/DragFileUploadDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react'
+import DialogBox from '~/components/templates/DialogBox'
+import FileUploadImage from '~/shared/icons/FileUploadImage'
+import { useDropzone } from 'react-dropzone'
+import { useAppSelector } from '~/hooks/reduxSelector'
+import toast from 'react-hot-toast'
+
+type Props = {
+  isOpen: boolean
+  isSubmitting: boolean
+  closeModal: () => void
+  actions: {
+    handleUploadFiles: (data: any, callback: () => void) => Promise<void>
+    onClick: () => void
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  }
+  children: React.ReactNode
+}
+
+const DragFileUploadDialog: FC<Props> = (props): JSX.Element => {
+  const error = useAppSelector((state) => state.file.error)
+  const isError = useAppSelector((state) => state.file.isError)
```

Comment:
```suggestion
  const {error, isError} = useAppSelector((state) => state.file)
```

Reply:
Collapsed these lines into a single line. Thank you!